tools/doxygen/Makefile

```diff
@@ -1,8 +1,9 @@
-contiking := ../..
-basedirs	:= os arch
-empty		:=
-space		:= $(empty) $(empty)
-pwd			:= $(shell cd $(contiking); pwd)
+contiking    := ../..
+basedirs	 := os arch
+empty		 :=
+space		 := $(empty) $(empty)
+pwd			 := $(shell cd $(contiking); pwd)
+exclude_dirs := "$(contiking)/arch/cpu/arm/CMSIS"
```

Comment:
Any reason not to use `EXCLUDE_PATTERNS` in `tools/doxygen/Doxyfile` instead?

```
EXCLUDE_PATTERNS       = */cpu/cc26x0-cc13x0/lib/* \
                         */cpu/cc26x0-cc13x0/rf-core/api/* \
                         */cpu/simplelink-cc13xx-cc26xx/lib/* \
                         */cpu/simplelink-cc13xx-cc26xx/rf-settings/* \
                         */cpu/simplelink-cc13xx-cc26xx/cc13x0-cc26x0/driverlib/* \
                         */cpu/simplelink-cc13xx-cc26xx/cc13x0-cc26x0/rf_patches/* \
                         */cpu/nrf52840/lib/* \
                         */platform/stm32nucleo-spirit1/stm32cube-lib/* \
                         */os/net/security/tinydtls/*
```

Reply:
Did you understand the issue I faced here? :)

Reply:
Not really, no 😁 But I have not tried either

Reply:
:D 
So the bash script collects the files to be passed to the Doxygen and saves it a env. variable. 
However these variables have a length limit.
I was unlucky and with the CMSIS we hit the ceiling. :) 
So I had to filter in the bash script.